vms/platformvm/block/builder/builder.go

```diff
@@ -355,6 +355,9 @@ func buildBlock(
 		)
 	}
 	if err != nil {
+		builder.txExecutorBackend.Ctx.Log.Warn("failed to pack block transactions",
+			zap.Error(err),
+		)
```

Comment:
Note: engine will log at debug, but not warn level since block build failures can happen regularly if the VM cleared its mempool and decides not to produce an empty block.